docs/feature_unicode.md

```diff
@@ -97,12 +101,10 @@ You can switch the input mode at any time by using one of the following keycodes
 |`UNICODE_MODE_BSD`     |`UC_M_BS`|`UC_BSD`     |Switch to BSD input (not implemented).   |
 |`UNICODE_MODE_WINC`    |`UC_M_WC`|`UC_WINC`    |Switch to Windows input using WinCompose.|
 
-You can also switch the input mode by calling `set_unicode_input_mode(x)` in your code, where _x_ is one of the above input mode constants (e.g. `UC_LNX`). Since the function only needs to be called once, it's recommended that you do it in `eeconfig_init_user` (or a similar function). For example:
+To explicitly change the mode and update the EEPROM in code, call `set_unicode_input_mode(x)` with one of the input modes. To lock the mode to a fixed value in your keymapor keyboard, simply enable input method cycling with only one mode available, e.g. in a `config.h` add
 
-```c
-void eeconfig_init_user(void) {
```

Comment:
This shouldn't be removed, actually.  

It sets the default mode if the eeprom is reset.  
And when you reset the eeprom, it doesn't re-run the init code.  So .... this is probably the better way.  ESPECIALLY, if you want to use multiple methods (but want a default, still). 

But these are not mutually exclusive. options.